src/main/java/org/example/Main.java

```diff
@@ -17,83 +13,117 @@
-                    String dosage_form = products.getAsJsonArray().get(0).getAsJsonObject().getAsJsonPrimitive("dosage_form").getAsString();
-                    if (brandName == null) {
-                        continue;
-                    }
-                    brandNameList.add(brandName);
-
-                    Medication medication = new Medication();
-                    medication.setBrand_name(brandName);
-                    medication.setStrength(strength);
-                    medication.setDosage_form(dosage_form);
-                    medications.add(medication);
-
-                    statement.setString(1, brandName);
-                    statement.setString(2, strength);
-                    statement.setString(3, dosage_form);
-                    statement.execute();
-//                    for (JsonElement brandNameElement : brandName) {
-//                       // String brandName = brandNameElement.getAsString();
-//
-//                        Medication medication = new Medication();
-//                      //  medication.setBrand_name(brandName);
-//                        medications.add(medication);
-//
-//                       // statement.setString(1, brandName);
-//                       // statement.execute();
-//                    }
-                }
-                for (String brand : brandNameList) {
-                    System.out.println(brand);
-                }
-                for(Medication medication : medications){
-                    System.out.println(medication.toString());
-                }
-
-           statement.close();
-            jDbCconnection.close();
-        } else {
-            System.out.println("Error retrieving medication data. Response code: " + responseCode);
+  public static void main(String[] args) throws Exception {
+
+    String apiUrl = "https://api.fda.gov/drug/drugsfda.json?&limit=1000";
+
+    URL url = new URL(apiUrl);
+    HttpURLConnection connection = (HttpURLConnection) url.openConnection();
+    connection.setRequestMethod("GET");
+
+    int responseCode = connection.getResponseCode();
+    if (responseCode == HttpURLConnection.HTTP_OK) {
+      BufferedReader in = new BufferedReader(new InputStreamReader(connection.getInputStream()));
+      String inputLine;
+      StringBuffer response = new StringBuffer();
+      while ((inputLine = in.readLine()) != null) {
+        response.append(inputLine);
+      }
+      in.close();
+
+      JsonObject jsonObject = new Gson().fromJson(response.toString(), JsonObject.class);
+      JsonArray results = jsonObject.getAsJsonArray("results");
+
+      Class.forName("org.postgresql.Driver");
+
+      Connection jDbCconnection =
+          DriverManager.getConnection("jdbc:postgresql://localhost:5432/dex");
+      PreparedStatement statement =
+          jDbCconnection.prepareStatement(
+              "INSERT INTO medication_with_uuid (brand_name, strength, dosage_form) VALUES (?,?,?)");
+
```

Comment:
i was able to add an extension to my test database that allows me to generate uuids with every row. it works! looks pretty offish.